threads/main.go

```diff
@@ -0,0 +1,210 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"github.com/c-bata/go-prompt"
+	ma "github.com/multiformats/go-multiaddr"
+	"github.com/textileio/go-threads/api/client"
+	"github.com/textileio/go-threads/util"
+	"google.golang.org/grpc"
+	"log"
+	"os"
+	"strings"
+	"time"
+)
+
+var (
+	apiClient      *client.Client
+	apiTimeout     = time.Second * 5
+	streamTimeout  = time.Second * 30
+	noStoreMessage = "Store required. `use <store id>`"
```

Comment:
totally, was hoping we could do something like that. the obvious way would be to construct a dynamic suggestions object from the list. we could provide descriptions for each, perhaps store name is the primary key and description contains the store id, that would allow for autocomplete too. or you could leave the description off which would make it a bit tidier is all. 

perhaps then it would be

`use <store name>`

`use -id=<store id>`

ome way to make it select by names as default and ids as a specific method. or other way around.